packages/mobile/src/identity/verification.ts

```diff
@@ -881,8 +881,32 @@ export function* reportRevealStatusSaga({
   issuer,
   pepper,
 }: ReportRevealStatus) {
+  let aggregatedResponse: undefined | { ok: boolean; status: number; body: any }
   if (shouldUseProxy()) {
-    // TODO We need to set proxy for this
+    Logger.debug(
+      `${TAG}@reportRevealStatusSaga`,
+      `Posting to proxy for service url ${attestationServiceUrl}`
+    )
+    const fullUrl = attestationServiceUrl + '/get_attestations'
+    const body = {
+      attestationServiceUrl: fullUrl,
+      account,
+      phoneNumber: e164Number,
+      issuer,
+      pepper,
+    }
+    try {
+      const proxyReveal = functions().httpsCallable('proxyRevealStatus')
+      const response = yield call(proxyReveal, body)
+      const { status, data } = response.data
+      const ok = status >= 200 && status < 300
+      aggregatedResponse = { ok, status, body: JSON.parse(data) }
+    } catch (error) {
+      Logger.error(`${TAG}@reportAttestationRevealStatus`, 'Error calling proxyRevealStatus', error)
+      // The httpsCallable throws on any HTTP error code instead of
+      // setting response.ok like fetch does, so catching errors here
+      aggregatedResponse = { ok: false, status: 500, body: error }
```

Comment:
interesting -- does the error code relate to the appropriate status code?  rather than hardcoding it to a 500 status response here?